apps/frontend/src/shared-research/__tests__/Routes.test.tsx

```diff
@@ -104,8 +59,10 @@ describe('the library page', () => {
     userEvent.click(checkbox);
     expect(checkbox).toBeChecked();
     await waitFor(() => {
-      const { searchParams } = new URL(req!.path, 'http://api');
-      expect(searchParams.get('filter')).toBe('Proposal');
+      expect(mockGetResearchOutputs).lastCalledWith(
```

Comment:
we should have a lint rule or something to make it clear, but we have used this style exactly zero times in the codebase, so please stick with `toHaveBeenLastCalledWith` :D 
(and below)